src/nft/hooks/useLastWallet.ts

```diff
@@ -0,0 +1,22 @@
+import create from 'zustand'
```

Comment:
This is probably no longer needed. Wallet management is handled by the global app.